core/src/test/scala/unit/kafka/server/KafkaServerTest.scala

```diff
@@ -42,6 +43,24 @@ class KafkaServerTest extends QuorumTestHarness {
     TestUtils.shutdownServers(Seq(server1, server2))
   }
 
+  @Test
+  def testListenerPortAlreadyInUse(): Unit = {
+    val serverSocket = new ServerSocket(0, 0, InetAddress.getLoopbackAddress)
+
+    var kafkaServer : Option[KafkaServer] = None
+    try {
+      TestUtils.waitUntilTrue(() => serverSocket.isBound, "Server socket failed to bind.")
+      // start a server with listener on the port already bound
+      assertThrows(classOf[RuntimeException],
+        () => kafkaServer = Option(createServerWithListenerOnPort(serverSocket.getLocalPort)),
+        "Exepected RuntimeException because of KafkaServer startup failure due do address already in use"
```

Comment:
nit: typo for `Expected`, `due to`. And the sentence could be rephrased, maybe:
"Expected RuntimeException due to address already in use when KafkaServer startup"

Reply:
fixed and rephrased, thanks